demo-docs/Makefile

```diff
@@ -23,13 +23,13 @@
 
 .EXPORT_ALL_VARIABLES:
 
-DOCS_DIR      := docs
+DOCS_DIR      := demo-docs
```

Comment:
That might be the only thing which might create a woe when used within downstream repositories?

Reply:
We might want to add a bit of heuristics to discover the "docs" directory from a list of `[docs, doc, demo-docs]`?

Reply:
oh good point, yeah

Reply:
Maybe I was overthinking this back then. `demo-docs/Makefile` will actually be just a blueprint, so there will be no need to provide a directory autodiscovery here anyway, right?